.gitignore

```diff
@@ -4,3 +4,4 @@
 .pub/
 build/
 doc/api/
+.vscode/
```

Comment:
I think this is best to be gitignore'd globablly by the user instead of per project

Reply:
sure